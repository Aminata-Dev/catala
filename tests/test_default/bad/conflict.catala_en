## Article

```catala
declaration scope A:
  context x content integer

scope A:
  definition x under condition true consequence equals 1
  definition x under condition true consequence equals 0
```

```catala-test-inline
$ catala Interpret -s A
[WARNING] In scope "A", the variable "x" is never used anywhere; maybe it's unnecessary?

┌─⯈ tests/test_default/bad/conflict.catala_en:5.11-5.12:
└─┐
5 │   context x content integer
  │           ‾
  └─ Article
[ERROR] There is a conflict between multiple valid consequences for assigning the same variable.

This consequence has a valid justification:
┌─⯈ tests/test_default/bad/conflict.catala_en:8.56-8.57:
└─┐
8 │   definition x under condition true consequence equals 1
  │                                                        ‾
  └─ Article

This consequence has a valid justification:
┌─⯈ tests/test_default/bad/conflict.catala_en:9.56-9.57:
└─┐
9 │   definition x under condition true consequence equals 0
  │                                                        ‾
  └─ Article
#return code 255#
```
