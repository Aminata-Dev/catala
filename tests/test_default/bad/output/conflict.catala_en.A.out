[ERROR] There is a conflict between multiple exceptions for assigning the same variable.

This justification is true:
 --> test_default/bad/conflict.catala_en
  | 
9 |   definition x under condition true consequence equals 0
  |                                ^^^^
  + Article

This justification is true:
 --> test_default/bad/conflict.catala_en
  | 
8 |   definition x under condition true consequence equals 1
  |                                ^^^^
  + Article
